lib/interferon.rb

```diff
@@ -65,10 +65,10 @@ def read_alerts
       abort("no such directory #{path} for reading alert files") \
         unless Dir.exists?(path)
 
-      Dir.glob(File.join(path, '*.rb')) do |alert_file|
+      Dir.glob(File.join(path, '**/*.rb')) do |alert_file|
         break if @request_shutdown
         begin
-          alert = Alert.new(alert_file)
+          alert = Alert.new(@alerts_repo_path, alert_file)
```

Comment:
`glob` returns relative paths; so, you'll get, like, `alert_name.rb` and also `subdir/alert_name.rb`. based on that, it seems sufficient to still just pass `alert_file`.